RELEASE_NOTES.md

```diff
@@ -1,3 +1,173 @@
+# ChefDK 4.5
+
+## Updated Components
+
+### Chef Infra Client
+
+Chef Infra Client has been updated from 15.3 to 15.4 with updated resources and several significant fixes to `knife bootstrap`. See the [Chef Infra Client 15.4 Release Notes](https://discourse.chef.io/t/chef-infra-client-15-4-45-released/16081) for a complete list of the new and improved functionality.
+
+### Chef InSpec
+
+Chef InSpec has been updated from 4.16 to 4.18 with the following changes:
+
+#### New Features
+
+- Use Waivers to mark controls as being administratively expected to fail.
+- We have released our beta Chef InSpec plug-in for HashiCorp Vault. Check it out in our inspec-vault GitHub repo and let us know what you think -- or better yet, start jumping in and contributing with us on it.
+- Waivers, our new beta feature, was added to InSpec! Waivers allows you to better manage compliance failures. We would love to hear your feedback on this! See our documentation for more details.
```

Comment:
Did you want to include a link for the documentation...? 😬

Reply:
Added